models/user.go

```diff
@@ -131,14 +131,21 @@ type User struct {
 	// Maximum repository creation limit, -1 means use global default
 	MaxRepoCreation int `xorm:"NOT NULL DEFAULT -1"`
 
-	// Permissions
-	IsActive                bool `xorm:"INDEX"` // Activate primary email
-	IsAdmin                 bool
-	IsRestricted            bool `xorm:"NOT NULL DEFAULT false"`
+	// IsActive true: primary email is activated, user can access Web UI and Git SSH.
```

Comment:
`activated` can be vague, I think `verified` can be used instead.
```suggestion
	// IsActive true: primary email is verified, user can access the Web UI and Git SSH.
```

Reply:
We should keep "activated", this concept is used widely in Gitea code base.

Reply:
That makes sense.